lib/rucio/core/rse.py

```diff
@@ -757,7 +753,6 @@ def get_rse_protocols(rse, schemes=None, session=None):
             'rse_type': str(_rse.rse_type),
             'credentials': None,
             'volatile': _rse.volatile,
-            'verify_checksum': verify_checksum[0] if verify_checksum else True,
```

Comment:
hmmmm, did you remove this on purpose? i thought frank put this explicitly to make webdav transfers work.

Reply:
Oups, no. Will put it back

Reply:
also, line 742 above, just saw it